llbuild.xcodeproj/xcshareddata/xcbaselines/E1C404AB1A0308F3003392BA.xcbaseline/80704D72-AD86-42E5-99D3-DB74CDEA1F5C.plist

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm, this was not supposed to be added.

Reply:
Well, it was not intentionally added.  But I do see several other baselines in the repo already.  I'm ambivalent as to whether we should or should not have these here.

Reply:
Lets not have them, and gitignore them